FreeAPS/Sources/Modules/NightscoutConfig/View/NightscoutFetchView.swift

```diff
@@ -0,0 +1,35 @@
+
+import SwiftUI
+
+struct NightscoutFetchView: View {
+    @ObservedObject var state: NightscoutConfig.StateModel
+
+    var body: some View {
+        Form {
+            Section {
+                Toggle("Fetch Treatments", isOn: $state.isDownloadEnabled)
+                    .onChange(of: state.isDownloadEnabled) { newValue in
+                        if !newValue {
+                            state.allowAnnouncements = false
+                        }
+                    }
+            } header: {
+                Text("Allow Fetching from Nightscout")
+            } footer: {
+                Text(
+                    "The Fetch Treatments toggle enables fetching of carbs and temp targets entered in Careportal or by another uploading device than Trio."
+                )
+            }
+            Section {
+                Toggle("Remote Control", isOn: $state.allowAnnouncements)
+                    .disabled(!state.isDownloadEnabled)
+            } header: { Text("Allow Remote control of Trio")
+            } footer: {
+                Text(
+                    "Fetch Treatments needs to be allowed to be able to toggle on Remote Control.\n\nWhen enabled you allow these remote functions through announcements from Nightscout:\n • Suspend/Resume Pump\n • Opening/Closing Loop\n • Set Temp Basal\n • Enact Bolus."
```

Comment:
I‘d suggest splitting these multiple texts into several text elements. Will only comment this for this LOC, applicable for the other info texts as well.

Reply:
Good point! You mean something like this?:
```swift
footer: VStack(alignment: .leading, spacing: 2) {
                        Text(
                            "Importing settings from Nightscout will overwrite these settings in Trio Settings -> Configuration:"
                        )
                        Text("• DIA (Pump settings)")
                        Text("• Basal Profile")
                        Text("• Insulin Sensitivities")
                        Text("• Carb Ratios")
                        Text("• Target Glucose")
                    }
```

Reply:
Yes exactly what I meant. You might also be able to use string literals here so that you can make use of the existing translations for all those setting (section) names 🤔

Reply:
Will that work with my pretty bullets • ? Hm... I guess I could make HStacks for each line and put the bullet in separat text elements (since the bullet don't needs to be translated)

Reply:
You should be able to either do something like 

`Text("• " + NSLocalizedString("…"))` or you can also concatenate multiple texts into one by doing 

`Text("…") + Text("…")`

Either one should definitely work with your lovely bullet points 😊

Reply:
Great! Also realized that I don't need to use the NSLocalizedString("xxx", comment: yyy) stuff to make translations work for text within footers/headers. However. Not all text have existing translations as is (And should of course be added, but I really need help from someone who knows how to efficiently update alla localization files without having to copy+paste text 100s of times)

Will make all text split changes and push a new commit in a little while.

Reply:
My understanding is that you need to use it when you are using string literals (so stuff like `"\(myvar)"` and when concatenating strings within a single `Text("…")`. The way you did it now should be fine without the NSLocalized 👍😊